### PR TITLE
feat: navigation component

### PR DESCRIPTION
**Pull request by sks-21**
Added **Navigation** component
Includes

- Simple navigation
- Navigation with the search bar
- Navigation with user

### CODE REVIEW

File: components/navigation/navigation.html [hunk 1]
```diff
@@ -140,6 +140,21 @@
...
                 <div class="content-heading-md">Navigation with search bar</div>
+                <nav class="navbar-1 flx-row flx-main-center">
+                    <div class="navbar-1-title">Navbar Heading</div>
+                    <div class="form-comp-1 flx-row flx-center nav-search">
+                        <input type="text" id="std-input" class=" form-input form-input-text" placeholder="Search"><span
+                            class="form-icon form-icon-search"><i class="fas fa-search"></i></span>
+                    </div>
+
+                    <div class="navbar-1-links">
+                        <ul class="flx-row flx-wrap navbar-1-ul">
```
Comment (rkapoor10) [inline components/navigation/navigation.html:151]: you may change the class name for flex properties

File: components/navigation/navigation.html [hunk 2]
```diff
@@ -0,0 +1,161 @@
...
+    <link rel="stylesheet" href="/components/navigation/navigation.css" />
+</head>
+
+<body>
+    <!-- || Container  -->
+    <div class="container">
+        <!-- || Navbar starts -->
+        <nav class="navbar flx-row flx-main-center">
+            <div class="navbar-logo">
+                <img src="/assets/images/futbolt-lg.png" alt="" loading="eager" />
```
Comment (rkapoor10) [inline components/navigation/navigation.html:19]: alt prop is empty

**Review by rkapoor10 — APPROVED** [review]
The code looks good to merge. Go through the suggestions once.

File: components/navigation/navigation.html (hunk 2 above)
Comment (Priya-Patel31) [inline components/navigation/navigation.html:19]: Give some alternate text in alt attribute

File: components/navigation/navigation.html [hunk 3]
```diff
@@ -0,0 +1,161 @@
...
+                    <a href="/components/toast/toast.html">Toast</a>
+                </li>
+                <li>
+                    <a href="/components/grid/grid.html">Simple Grid</a>
+                </li>
+            </ul>
+        </aside>
+
+        <!-- || Main content starts  -->
+        <main class="content flx-col">
```
Comment (Priya-Patel31) [inline components/navigation/navigation.html:94]: just a suggestion to use flex rather than flx

**Review by Priya-Patel31 — APPROVED** [review]
Everything looks good . Go ahead and merge it !!

**Pull request merged**